go/kbfs/libkbfs/folder_branch_status.go

```diff
@@ -48,6 +48,8 @@ type FolderBranchStatus struct {
 	Unmerged []*crChainSummary
 	Merged   []*crChainSummary
 
+	ConflictResolutionAttempts []conflictRecord
```

Comment:
Also it's a bit weird to have expose an unexported struct in an exported struct field like this, so that might mean it makes sense to export `conflictRecord`.  I don't have a strong feeling about it though.

Reply:
the `crChainSummary`s above are also unexported, and it seemed fine to me.